src/js/preview.js

```diff
@@ -36,23 +26,28 @@ $(function () {
     });
   }
 
-  // TODO
-  function captureTabScreen(tabId, screenImg) {
-    chrome.tabs.onActivated.addListener(function () {
-      chrome.tabs.captureVisibleTab(function (capturedUrl) {
-        screenImg.attr('src', capturedUrl);
-      })
-    });
+  function addNewTab(tabInfo) {
+    var newTabDiv = tabDiv.clone();
```

Comment:
ditto, please consider about using createElement.(see http://stackoverflow.com/questions/494143/creating-a-new-dom-element-from-an-html-string-using-built-in-dom-methods-or-pro)